index.js

```diff
@@ -96,6 +96,7 @@ let executeHooksWithArgs = (hooks = [], args) => {
                 commandIsRunning = _commandIsRunning
             }).run()
         } catch (e) {
+            reject(e)
```

Comment:
I am not sure if this is a good idea, I don't want that any services can blow up the test suite if they don't work. That is way I went without calling reject.